dev/customer_testing/lib/customer_test.dart

```diff
@@ -24,8 +24,6 @@ class CustomerTest {
         contacts.add(line.substring(8));
       } else if (line.startsWith('fetch=')) {
         fetch.add(line.substring(6));
-      } else if (line.startsWith('update=')) {
```

Comment:
If you remove this whole case, then the `FormatException` below will be thrown, and I'm guessing the customer test presubmit checks will fail. I think we'd want to continue to allow the `update` field until the field in the files in `flutter/tests` can be removed.

Reply:
Thanks! Done, and I linked to the upstream script that needs to be implemented before we enable this again.